services/ux-backend/handlers/constants.go

```diff
@@ -0,0 +1,8 @@
+package handlers
+
+// this file contains that constants that are common across all handlers
+
+const (
+	ContentTypeTextPlain                    = "text/plain"
+	OnboardingValidationPublicKeySecretName = "onboarding-ticket-key"
```

Comment:
Why do we need this here? Do we have multiple handlers using this constant?

Reply:
my mistake, missed it.